frontend/src/composables/useShootEditor/index.js

```diff
@@ -68,36 +102,31 @@ export function useShootEditor (initialValue, options = {}) {
   })
 
   let editorLineHighlighter = null
+  let initialDocumentValue = null
 
   const schemaDefinition = computedAsync(() => {
     return api.getShootSchemaDefinition()
   }, null)
 
-  const defaultExtraKeys = {
-    Tab: instance => {
-      if (instance.somethingSelected()) {
-        instance.indentSelection('add')
-      } else {
-        instance.execCommand('insertSoftTab')
-      }
-    },
-    'Shift-Tab': instance => {
-      instance.indentSelection('subtract')
+  const defaultExtraKeys = [
+    {
+      key: 'Enter',
+      run: view => {
+        completions.value?.editorEnter(view)
+        return true
+      },
     },
-    Enter: instance => {
-      completions.value?.editorEnter(instance)
-    },
-    'Ctrl-Space': 'autocomplete',
-  }
+  ]
 
   function getExtraKeys () {
-    const originalExtraKeys = {
+    const extraKeys = [
       ...defaultExtraKeys,
-      ...options.extraKeys,
-    }
-    const extraKeys = {}
-    for (const [key, value] of Object.entries(originalExtraKeys)) {
-      extraKeys[localStorageStore.editorShortcuts[key] ?? key] = value
+      ...(options.extraKeys ?? []),
+    ]
+    for (const extraKey of extraKeys) {
+      if (localStorageStore.editorShortcuts[extraKey.key]) {
+        extraKey.key = localStorageStore.editorShortcuts[extraKey.key]
```

Comment:
use lodash get instead of using bracket notation

```js
if (get(localStorageStore.editorShortcuts, [extraKey.key])) {
    extraKey.key = get(localStorageStore.editorShortcuts, [extraKey.key])
}
```